src/database/models/article.js

```diff
@@ -0,0 +1,50 @@
+module.exports = (sequelize, DataTypes) => {
+  const Article = sequelize.define('Article', {
+    title: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        len: [2]
+      }
+    },
+    description: DataTypes.STRING,
+    content: DataTypes.TEXT,
+    image: DataTypes.STRING,
+    slug: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      unique: true
+    },
+    publisheddate: {
+      type: DataTypes.DATE,
+      field: 'published_date',
+      defaultValue: new Date()
+    },
+    status: {
+      type: DataTypes.ENUM('draft', 'active', 'deactivated'),
+      // type: DataTypes.ENUM('DRAFT', 'ACTIVE', 'DEACTIVATED'),
+      defaultValue: 'draft'
+    },
+    createdAt: {
+      type: DataTypes.DATE,
+      allowNull: false,
+      field: 'created_at',
+      defaultValue: new Date()
+    },
+    updatedAt: {
+      type: DataTypes.DATE,
+      allowNull: false,
+      field: 'updated_at',
+      defaultValue: new Date()
+    }
+  });
+
+  Article.associate = models => {
+    // associations can be defined here
```

Comment:
Also remove this

Reply:
Thanks for this review.
Can you point explicitly to what you are referring to as **`this`**

Reply:
I think she meant the comment. Let's ensure all unused variables and stubs are removed.

Reply:
Acknowledged. I will look into it.